### ISSUE TITLE
Rethink navigation/UI, including mobile

### ISSUE THREAD

**Issue by jacobobryant** [jacobobryant/platypub#42]
After #39 is closed, add a link below Posts for each site you've created:

```
[Posts]
├── [My Site]
└── [Another Site]
[Sites]
[Newsletters]
```

The link could look something like `/app?site-id=...`, i.e. we just pass in a query param to the posts page and use it to filter the posts shown. (that also reminds me we should fix that `/app` so it's just `/posts` or something... should make a separate issue for that)

**Comment by jacobobryant** [comment]
Perhaps we should wait until #43 is closed, since we'll want to update the navigation once that's finished anyway.

**Comment by jacobobryant** [comment]
Now that platypub supports custom item types, I think the data model won't change too much more. It would be worth thinking about how to structure Platypub's UI.
